tests/test_report.py

```diff
@@ -14,26 +15,43 @@ def __init__(self, directory, client):
         self.settings.Vcs.type = "github"
         self.settings.MainVcs.report_to_review = True
         self.settings.GitVcs.repo = client.server.url
-        commit_id = six.text_type(client.repo.remotes.origin.refs[client.server.target_branch].commit)
+        commit_id = str(client.repo.remotes.origin.refs[client.server.target_branch].commit)
         self.settings.GitMainVcs.checkout_id = commit_id
-        self.settings.GithubMainVcs.token = "token"
+        self.settings.GithubToken.integration_id = "1234"
+        self.settings.GithubToken.key = "configs.py"      # TODO: fix after proper implementing
+        self.settings.GithubTokenWithInstallation.installation_id = "5678"
         self.settings.GithubMainVcs.check_id = "123"
         self.settings.GithubMainVcs.api_url = "http://localhost/"
         self.settings.Reporter.report_start = True
         self.settings.Reporter.report_success = True
 
-        self.path = "http://localhost/repos" + \
-                    six.text_type(client.root_directory).rsplit("client", 1)[0] + \
-                    "server/check-runs/123"
+        repo_name = str(client.root_directory).rsplit("client", 1)[0]
+        self.path = "http://localhost/repos" + repo_name + "server/check-runs/123"
+
+
+@pytest.fixture()
+def mock_github_token(request):
```

Comment:
Please try to use monkeypatch in the test instead of implementing fixture on your own.
I am not sure whether it will look better, so we need to see how it looks.